web_transport/sdk/impl/web_transport_owt_client_impl.cc

```diff
@@ -150,8 +150,13 @@ void WebTransportOwtClientImpl::OnError(const net::WebTransportError& error) {
 }
 
 void WebTransportOwtClientImpl::OnClosed(
-      const absl::optional<net::WebTransportCloseInfo>& close_info) {
-  LOG(INFO) << "OnClosed.";
+    const absl::optional<net::WebTransportCloseInfo>& close_info) {
+  if (!visitor_) {
+    return;
+  }
+  visitor_->OnClosed(
+      close_info.has_value() ? close_info->code : 0,
```

Comment:
you should avoid calling close_info.has_value() twice.

Reply:
Done.